src/Spectre.libGenetic/Individual.cpp

```diff
@@ -29,77 +29,28 @@ using namespace spectre::core::exception;
 namespace spectre::algorithm::genetic
 {
 Individual::Individual(std::vector<bool> &&binaryData):
-    m_BinaryData(binaryData) { }
+    std::vector<bool>(binaryData) { }
 
 const std::vector<bool>& Individual::getData() const
 {
-    return m_BinaryData;
-}
-
-std::vector<bool>::reference Individual::operator[](size_t index)
-{
-    if (index < m_BinaryData.size())
-    {
-        return m_BinaryData[index];
-    }
-    else
-    {
-        throw OutOfRangeException(index, m_BinaryData.size());
-    }
-}
-
-std::vector<bool>::const_reference Individual::operator[](size_t index) const
-{
-    if (index < m_BinaryData.size())
-    {
-        return m_BinaryData[index];
-    }
-    else
-    {
-        throw OutOfRangeException(index, m_BinaryData.size());
-    }
-}
-
-std::vector<bool>::iterator Individual::begin()
-{
-    return m_BinaryData.begin();
-}
-
-std::vector<bool>::iterator Individual::end()
-{
-    return m_BinaryData.end();
-}
-
-std::vector<bool>::const_iterator Individual::begin() const
-{
-    return m_BinaryData.begin();
-}
-
-std::vector<bool>::const_iterator Individual::end() const
-{
-    return m_BinaryData.end();
-}
-
-size_t Individual::size() const
-{
-    return m_BinaryData.size();
+    return *this;
 }
 
 bool Individual::operator==(const Individual &other) const
```

Comment:
The [default operator of std::vector](https://stackoverflow.com/a/16422594) won't make a cut?

Reply:
Interestingly, I could get it to compile with custom `operator==` removed. But additionally removing `operator!=` causes a build failure in GTest.

Let’s leave both as is for now.